docs/standard/garbage-collection/fundamentals.md

```diff
@@ -24,28 +24,8 @@ ms.author: "ronpet"
   
 -   Provides memory safety by making sure that an object cannot use the content of another object.  
   
- This topic describes the core concepts of garbage collection. It contains the following sections:  
-  
--   [Fundamentals of memory](#fundamentals_of_memory)  
-  
--   [Conditions for a garbage collection](#conditions_for_a_garbage_collection)  
-  
--   [The managed heap](#the_managed_heap)  
-  
--   [Generations](#generations)  
-  
--   [What happens during a garbage collection](#what_happens_during_a_garbage_collection)  
-  
--   [Manipulating unmanaged resources](#manipulating_unmanaged_resources)  
-  
--   [Workstation and server garbage collection](#workstation_and_server_garbage_collection)  
-  
--   [Concurrent garbage collection](#concurrent_garbage_collection)  
-  
--   [Background workstation garbage collection](#background_garbage_collection)  
-  
--   [Background server garbage collection](#background_server_garbage_collection)  
-  
+ This topic describes the core concepts of garbage collection. 
+ 
```

Comment:
why are these links removed?

Reply:
They're the mini-TOC that was at the top of the topic, @Maoni0. We've been removing when we've been touching articles, since there's now an automatically generated TOC on the right side of the article that remains in view even for scrollable articles.

Reply:
got it, makes sense!